docs/user-guide/configure-xmem-server.md

```diff
@@ -1,7 +1,7 @@
-# Configuring the Zowe cross memory server (ZIS)
+# (Optional) Configuring the Zowe cross memory server (ZIS)
```

Comment:
what do you mean by optional here?
if you mean that zowe can be configured to not use this, then you should write optional on every single part of zowe server configuration because zowe can be configured to run with any combination of disabled components.
but putting optional on every section would just be confusing.

if thats the case, instead of saying optional here, you could just say that ZIS is only needed when the components zss or app-server are used (parts of the app framework) or if an extension requires it or them.